docs/dietpi_tools/system_maintenance.md

```diff
@@ -67,53 +67,69 @@ dietpi-backup
+    The filter definition syntax is described within the file itself.
 
-The automatic daily backup (activated via option "Daily Backup", see screenshot above) is controlled via the Linux `cron` mechanism. Setting a different starting time can be an option, e.g. if you have several backup clients backing up to the same storage (backup server): Shifting the backup starting time of these systems may reduce temporary overload of the backup server by avoiding concurrent access to the storage.
+    ![DietPi-Backup filter option screenshot](../assets/images/dietpi-backup_filter-option.jpg){: width="681" height="330" loading="lazy"}
 
-The starting time is basically defined via the file `/etc/crontab` (which calls the backup/restore function via the `/etc/cron.daily/dietpi` script). It can be changed via the entry `cron.daily` within [`dietpi-cron`](../system_configuration/#dietpi-cron). It is executed by running the following command
+    The file containing the filter definitions is `/boot/dietpi/.dietpi-backup_inc_exc`.
 
-```sh
-dietpi-cron
-```
+=== "Space check"
 
-Please keep in mind that all other daily `cron` based procedures are also started at this changed time.
+    A space check on the target location prior to the backup process can be enabled/disabled. This might be an option if there is surely enough disk space available.  
+    Enabling the space check makes the backup a bit more safe, disabling it speeds it up.
 
-### Backup file selection (Filter)
+=== "Logging"
 
-The definition which files are used for the backup procedure is defined via the option "Filter" (see screenshot above). This opens `nano` to edit the include/exclude definitions for the backup.  
-The filter definition syntax is described within the file itself.
+    Logging information about the backup procedure is given within the files `.dietpi-backup_stats` and `.dietpi-backup.log` which are located in the backup target directory ("Location" option):
 
-![DietPi-Backup filter option screenshot](../assets/images/dietpi-backup_filter-option.jpg){: width="681" height="330" loading="lazy"}
+    - `.dietpi-backup_stats` gives a list of completed operations with time and date
+    - `.dietpi-backup.log` gives a list of every processed file
 
-The file containing the filter definitions is `/boot/dietpi/.dietpi-backup_inc_exc`.
+### Location of the backup
 
-### Space check
+The location of the backup can be set to any directory which is not part of the backup itself. Especially, these options are typical:
 
-A space check on the target location prior to the backup process can be enabled/disabled. This might be an option if there is surely enough disk space available.  
-Enabling the space check makes the backup a bit more safe, disabling it speeds it up.
+- Backup to/Restore from a connected USB stick or USB disk
+- Backup to/Restore from a large system disk mounted below `/media` (because `/media` is not backed up by default)
```

Comment:
```suggestion
```
This is probably redundant in combination with the first option. For clarification, also about "which is not part of the backup itself": `dietpi-backup` adds the target location automatically to excludes, so one can really chose anything as backup location and a sync loop is ruled out. The only limitation is that the target filesystem needs to support UNIX permissions and symlinks.